DominicanWhoCodes.Blazor/wwwroot/data/developers.json

```diff
@@ -134,6 +134,15 @@
     "linkedin": "www.linkedin.com/in/AVTO",
     "github": "https://github.com/Avto9711",
     "twitter":"https://twitter.com/angeliisimo"
+  },
+  {
+    "name": "Albert Eduardo Hidalgo Taveras",
+    "initials": "AEHT",
+    "image": "data/devs-pics/albert-hidalgo.jpg",
+    "summary": "Super moto and coding",
+    "skills": "PHP (Symfony, Ligne[Own Framework]), Vue, UX",
```

Comment:
Check this out, I think you missed a ")" after "(Symphony".

Fix this and resolve the conflicts that will generate the merge I'm about to merge (#40) from @cvivieca